autoconfig/install.sh

```diff
@@ -14,4 +14,12 @@ cp $AUTOCONFIG_DIR/nginx_config /etc/nginx/sites-enabled/autoconfig_and_autodisc
 sed -i "s#__EASYMAIL_SSL_CA_BUNDLE_FILE__#$SSL_CA_BUNDLE_FILE#g" /etc/nginx/sites-enabled/autoconfig_and_autodiscover
 sed -i "s#__EASYMAIL_SSL_PRIVATE_KEY_FILE__#$SSL_PRIVATE_KEY_FILE#g" /etc/nginx/sites-enabled/autoconfig_and_autodiscover
 
-service nginx reload
\ No newline at end of file
+service nginx reload
+
+# Set up the bash script for autostarting the services in case of reboot
+cp $AUTOCONFIG_DIR/run.sh /run.sh
```

Comment:
This file is for mail clients auto configuration:
- Autodiscover for Microsoft Outlook 
- Autoconfiguration for Thunderbird https://developer.mozilla.org/en-US/docs/Mozilla/Thunderbird/Autoconfiguration

It's should be in other directory something like "autostart"